core/block/import/converter/filenameprovider.go

```diff
@@ -0,0 +1,67 @@
+package converter
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/anyproto/anytype-heart/pkg/lib/core"
+	oserror "github.com/anyproto/anytype-heart/util/os"
+)
+
+func ProvideFileName(fileName string, files map[string]io.ReadCloser, path string, tempDirProvider core.TempDirProvider) (string, bool, error) {
```

Comment:
This code was added to HTML import here https://github.com/anyproto/anytype-heart/pull/348. But I move this function to common directory, because it's used not only by HTML import, but also by Markdown import